src/components/Calculator.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import Key from './Key';
+import Output from './Output';
+import Handler from './Handler';
+
+class Calculator extends React.Component {
+  render() {
+    return <div className="grid container max-width-sm">
+      <Output value={0} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'AC'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'+/-'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'%'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md foo'} name={'÷'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'7'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'8'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'9'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md foo'} name={'x'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'4'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'5'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'6'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md foo'} name={'-'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'1'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'2'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'3'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md foo'} name={'+'} />
+      <Key clickHandler={Handler} classes={'col-6 btn btn--subtle padding-y-md'} name={'0'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md'} name={'.'} />
+      <Key clickHandler={Handler} classes={'col-3 btn btn--subtle padding-y-md foo'} name={'='} />
```

Comment:
**Changes Required**
Good effort abstracting the button in its own `Class Component` as Key Component. However;
- [ ] There is too much repetition ongoing here, breaking our principle of `DRY`. Let us try and store the button names in an array. In this way, we can use `array.map` to dynamically add all the buttons to the DOM.